client/src/components/screens/Patient/components/FilesTab/test/files.spec.tsx

```diff
@@ -0,0 +1,247 @@
+                  "url": "https://ferload.qa.clin.ferlab.bio/48700352-aeec-4569-a053-f6fdc0baee08"
+                },
+                "format": "TGZ"
+              }
+            ],
+            "id": "DocumentReference/151719/_history/1",
+            "organization": {
+              "id": "Organization/OR00215",
+              "resource": {
+                "name": "LDM CHU Ste-Justine"
+              }
+            },
+            "task": {
+              "focus": {
+                "reference": "ServiceRequest/QA-SR-00002"
+              },
+              "id": "Task/151722/_history/1",
+              "runDate": "2020-11-06"
+            },
+            "type": "QC"
+          },
+          {
+            "aliquot": {
+              "id": [
+                "Specimen/152484"
+              ],
+              "resource": [
+                {
+                  "external_id": "16775",
+                  "organization": {
+                    "reference": "Organization/71615",
+                    "resource": {
+                      "name": "CQGC"
+                    }
+                  },
+                  "sample": [
+                    {
+                      "reference": "Specimen/152483",
+                      "resource": {
+                        "external_id": "NA24143_A",
+                        "organization": {
+                          "reference": "Organization/OR00215",
+                          "resource": {
+                            "name": "LDM CHU Ste-Justine"
+                          }
+                        }
+                      }
+                    }
+                  ]
+                }
+              ]
+            },
+            "content": [
+              {
+                "attachment": {
+                  "hash": "MzQ5NDc5OGViM2RmMDY4MWRiMmY2YzI0ZTJiMzdiYzY=",
+                  "title": "16775.cram",
+                  "url": "https://ferload.qa.clin.ferlab.bio/ad6cb890-991e-4cfe-85c8-12fb33db44ad"
+                },
+                "format": "CRAM"
+              },
+              {
+                "attachment": {
+                  "title": "16775.cram.crai",
+                  "url": "https://ferload.qa.clin.ferlab.bio/ad6cb890-991e-4cfe-85c8-12fb33db44ad.crai"
+                },
+                "format": "CRAI"
+              }
+            ],
+            "id": "DocumentReference/151717/_history/1",
+            "organization": {
+              "id": "Organization/OR00215",
+              "resource": {
+                "name": "LDM CHU Ste-Justine"
+              }
+            },
+            "task": {
+              "focus": {
+                "reference": "ServiceRequest/QA-SR-00002"
+              },
+              "id": "Task/151720/_history/1",
+              "runDate": "2020-11-06"
+            },
+            "type": "AR"
+          }
+        ],
+        "id": "Patient/QA-PA-00002/_history/4"
+    
+    }})
+    );
+
+        render(
```

Comment:
create a method for this since it used in the other test too and is a form of duplication.